packages/next-app/src/app/internalmanager/(components)/TokenSelect.tsx

```diff
@@ -239,15 +250,15 @@ function TokenRow({
       </Table.BodyCell>
       <Table.BodyCell>
         <Link
-          href={`/internalmanager/${chainName}/${operationKind}/${token.token_address}`}
+          href={`/internalmanager/${chainName}/${operationKind}/${token.tokenAddress}`}
         >
           <button
             type="button"
             onClick={() => {
               close?.();
             }}
           >
-            {token.balance}
+            {token.formatted}
```

Comment:
I assume this `formatted` comes from `FetchBalanceResult` type?

Reply:
asking because it's not a very descriptive name, but if it comes from wagmi types don't worry about this.

Reply:
yess it comes from wagmi types